src/Symfony/Component/Notifier/Bridge/AllMySms/AllMySmsOptions.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\AllMySms;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author gnito-org <https://github.com/gnito-org>
+ */
+final class AllMySmsOptions implements MessageOptionsInterface
+{
+    private array $options;
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    public function getAlerting(): ?int
```

Comment:
are getters needed?

Reply:
Just following the precedent that is set in `MessageInterface`.

Reply:
I meant MessageOptionsInterface.